recipes/towhee/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set name = "towhee" %}
+{% set version = "0.4.0rc1" %}
```

Comment:
Can you set this to a release? Otherwise you would need to add a config to deal with a pre-release channel.

Reply:
okey, let's will use 0.3.0 release first.